### DISCUSSION TITLE
The Servers Seem Down?

### DISCUSSION THREAD

**Discussion by koyuul** [lukePeavey/quotable#89] [Archived]
I can't access either the official api or the branch.

**Comment by itsSiddharthGupta** [comment]
True

**Comment by nickglovdev** [comment]
Happening to me as well. Trying to use it for project.

**Comment by lukePeavey** [comment]
Thanks for letting me know I'll look into this today!

**Comment by lukePeavey** [comment]
This issue has been resolved. Both API servers are up and running. Please let me know if things are working on your end now. I'll leave this issue open until I hear from you. 

Sorry for the inconvenience!

**Comment by itsSiddharthGupta** [comment]
Its working now

**Comment by lukePeavey** [comment]
I'm going to close this now. 

@kayul thanks for reporting this
